Add Checkpoints to 01_index (#472)
microsoft/rag-experiment-accelerator: README.md

    "openai_temperature": "determines the OpenAI temperature. Valid value ranges from 0 to 1.",
    "search_relevancy_threshold": "the similarity threshold to determine if a doc is relevant. Valid ranges are from 0.0 to 1.0",
    "chunking_strategy": "determines the chunking strategy. Valid values are 'azure-document-intelligence' or 'basic'",

>     "use_checkpoints": "A boolean to control Whether or not to use checkpoints to load data and skip processing that was already done in previous executions.",

Reviewer: ```suggestion
    "use_checkpoints": "A boolean. If true, enables use of checkpoints to load data and skip processing that was already done in previous executions.",
```
